runtime/src/xcm_config.rs

```diff
@@ -228,7 +242,7 @@ parameter_type_with_key! {
 pub struct AccountIdToMultiLocation;
 impl sp_runtime::traits::Convert<AccountId, MultiLocation> for AccountIdToMultiLocation {
     fn convert(account: AccountId) -> MultiLocation {
-        X1(AccountId32 { network: Some(NetworkId::Rococo), id: account.into() }).into()
+        X1(AccountId32 { network: None, id: account.into() }).into()
```

Comment:
Why not `Some(RelayNetwork::get())`?

Reply:
I changed it to more relaynetwork agnostic way, but yeah - I think it might be better to do it your way